CMakeLists.txt

```diff
@@ -165,9 +167,10 @@ if(TARGET Catch2::Catch2WithMain)
     get_filename_component(_testname "${_filename}" NAME_WE)
     add_executable("${_testname}" "${_filename}")
     target_link_libraries("${_testname}" PRIVATE MainTest)
+    target_compile_options("${_testname}" PRIVATE -O0 -g)
```

Comment:
Aren't these flags for GCC / Clang ? I think the MSVC ones are /O0 /O1 etc... and /DEBUG for debug info.
I'd recommend not having this at all in the cmakelists and just build with CMAKE_BUILD_TYPE=Debug which does the right thing for each compiler